servers/uchilahmc/metadata.json

```diff
@@ -0,0 +1,64 @@
+{
+    "id": "uchilahmc",
+    "name": "UchilahMC Network",
+    "description": "Minigames PartyGames, BedWars, BattleRoyale, Parkour e muito mais no UchilahMC",
+    "addresses": [
+        "uchilahmc.com"
+    ],
+    "primaryAddress": "uchilahmc.com",
+    "minecraftVersions": [
```

Comment:
Please update your supported versions. If it supports all versions, you do not also need to specify subversions.

For example, if you specify `1.19.*` you do not need to provide `1.19`, `1.19.2`, etc.

Reply:
Ok, I had taken it from the template and didn't even notice, I already made the changes